src/sass/_permission_groups.scss

```
@@ -140,16 +140,25 @@
 
     tr.strikeout {
       td {
+        opacity: 0.6;
         position: relative;
       }
```

The opacity shouldn't apply to the actions:

```suggestion
      td:not(.actions) {
        opacity: 0.6;
        position: relative;
      }
```